packages/dev/core/src/Rendering/IBLShadows/iblShadowsVoxelRenderer.ts

```diff
@@ -507,15 +511,17 @@ export class _IblShadowsVoxelRenderer {
         this._mipArray.forEach((mip) => {
             mip.dispose();
         });
+        this._voxelMaterial?.dispose();
+        this._voxelSlabDebugMaterial?.dispose();

```

Comment:
No need for the "?", as these properties are created in the constructor (several occurrences).

Reply:
Done